bin/clever.js

```diff
@@ -190,7 +190,7 @@ function run () {
       metavar: 'before',
       aliases: ['until'],
       parser: Parsers.date,
-      description: 'Fetch logs before this date/time (ISO8601 date or duration, positive number in seconds or duration Ex: 1h)',
+      description: 'Fetch logs before this date/time (ISO8601 date, positive number in seconds or duration, e.g.: 1h)',
```

Comment:
suggestion: can you also do this on `after`/`since`?

Reply:
It's ok and rebased